### PR TITLE
Measurement Input and UI updated

### PR DESCRIPTION
**Pull request by goelrohan6**
(https://cloud.githubusercontent.com/assets/14099191/13553552/64aa01c6-e3b3-11e5-8326-fc2589bc9f4c.png

)
screenshot from 2016-03-06 15 51 16

Issue #3  solved.
Given Jquery support to oprioins.js,
Added support for showing CO2 emissions in pounds if someone inputs units in pounds

- Fuel Consumption : {ltr/gln} / {km/mile}
- Fuel Efficiency : km/ltr, km/gl, mile/ltr
- Carbon Emission : {kg/pound/lb} / (km/mile)  

Also updated some UI.

### CODE REVIEW

File: Source/options.html [hunk 1]
```diff
@@ -1,38 +1,141 @@
...
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
         </div>
     </div>
+    <!-- /.container -->
+    <!-- jQuery -->
+    <script src="jquery.js"></script>
```
Comment (rohankatyal29) [inline Source/options.html]: you should use the minified version of jQuery

File: Source/options.js [hunk 2]
```diff
@@ -1,27 +1,129 @@
...
+                var fuelConsumption = 2328 * inputValue;
+                localStorage['carbonEmission'] = fuelConsumption;
+            } else {
+                var fuelConsumption = 2328 * inputValue;
+                localStorage['carbonEmission'] = fuelConsumption;
+            }
+            break;
+        case "g":
+            if (distance == "km") {
+                var fuelConsumption = 8782.155 * inputValue;
```
Comment (rohankatyal29) [inline Source/options.js]: Why are these values hardcoded?

File: Source/options.js [hunk 3]
```diff
@@ -1,27 +1,129 @@
...
-}
+$('#save-button-3').click(function() {
+
+    var fuel = document.getElementById('fuelInput3');
+    var fuelValue = fuel.value;
+    var distance = document.getElementById('distanceInput3');
+    var distanceValue = distance.value;
+    var input = document.getElementById('co2EmissionInput');
+    var inputValue = input.value;
+    var status = document.getElementById('save-message-3');
```
Comment (rohankatyal29) [inline Source/options.js]: The previous id is in camel case and this one has dashes, please use a consistent convention. Preferably use dashes (just a personal preference)

File: Source/options.html [hunk 4]
```diff
@@ -1,38 +1,141 @@
...
+
 <head>
-    <title>Google Maps Carbon Footprint Options</title>
-    <link rel="stylesheet" href="options.css" type="text/css" media="screen" />
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <meta name="author" content="">
+    <title>The Big Picture - Start Bootstrap Template</title>
```
Comment (rohankatyal29) [inline Source/options.html]: The title is from the template you have used

File: Source/options.js (hunk 2 above)
Comment (goelrohan6) [inline Source/options.js]: @rohankatyal29 made changes, thankyou!

File: Source/options.js (hunk 3 above)
Comment (goelrohan6) [inline Source/options.js]: @rohankatyal29 changed the case to camel case.

File: Source/options.html (hunk 1 above)
Comment (goelrohan6) [inline Source/options.html]: @rohankatyal29 i have made the changes and used jquery.min.js, thankyou!

File: Source/options.html (hunk 4 above)
Comment (goelrohan6) [inline Source/options.html]: @rohankatyal29 made the changes thankyou for spotting.

**Comment by rohankatyal29** [comment]
@goelrohan6: I have left some suggestions in the code, please have a look at them

**Comment by goelrohan6** [comment]
@rohankatyal29 i have made the required changes, thankyou.

**Pull request closed without merging**